crates/autopilot/src/database/auction.rs

```diff
@@ -108,4 +108,20 @@ impl Postgres {
         let id = database::auction::replace_auction(&mut ex, &data).await?;
         Ok(id)
     }
+
+    pub async fn most_recent_auction(&self) -> Result<Option<dto::AuctionWithId>> {
+        let _timer = super::Metrics::get()
+            .database_queries
+            .with_label_values(&["load_most_recent_auction"])
+            .start_timer();
+
+        let mut ex = self.pool.acquire().await?;
+        let (id, json) = match database::auction::load_most_recent(&mut ex).await? {
```

Comment:
The `solver_competition` table hopefully goes away soonish ™️  and we also have this data in the `auction_prices` table.

Reply:
> determine from which auction we want the prices and so a fetch all

You can do this:
```sql
SELECT token, price
FROM auction_prices
WHERE auction_id = (
    SELECT MAX(auction_id)
    FROM auction_prices
);
```